apps/console/src/features/identity-providers/components/wizards/enterprise-idp-create-wizard.tsx

```diff
@@ -829,6 +832,30 @@ export const EnterpriseIDPCreateWizard: FC<EnterpriseIDPCreateWizardProps> = (
 
     };
 
+    const resolveDocumentationLink = (): ReactElement => {    
+        if ( selectedProtocol === "saml" ) {
```

Comment:
Let's get this from the enum.
```suggestion
        if ( selectedProtocol === SupportedAuthProtocolTypes.SAML ) {
```

Reply:
Fixed with 249e320604fcef016aaffc5e2bb956c0cd74921f